packages/experimental/Menu/src/Menu/Menu.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { menuName, MenuProps } from './Menu.types';
+import { stagedComponent } from '@fluentui-react-native/framework';
+import { useMenu } from './useMenu';
+import { useMenuContextValue } from './useMenuContextValue';
+import { MenuProvider } from '../context/menuContext';
+
+export const Menu = stagedComponent((props: MenuProps) => {
+  const state = useMenu(props);
+  const contextValue = useMenuContextValue(state);
+
+  return (_rest: MenuProps, children: React.ReactNode) => {
+    const childrenArray = React.Children.toArray(children) as React.ReactElement[];
+
+    if (__DEV__) {
+      if (childrenArray.length === 0) {
+        // eslint-disable-next-line no-console
+        console.warn('Menu must contain at least one child');
+      }
+
+      if (childrenArray.length > 2) {
+        // eslint-disable-next-line no-console
+        console.warn('Menu must contain at most two children');
+      }
+    }
+
+    let menuTrigger = undefined;
+    let menuPopover = undefined;
+    if (childrenArray.length === 2) {
+      menuTrigger = childrenArray[0];
+      menuPopover = childrenArray[1];
+    } else if (childrenArray.length === 1) {
+      menuPopover = childrenArray[0];
+    }
```

Comment:
In this case, what is rendered on line 38?

Reply:
Just the popover. Since the first child is undefined, it shouldn't render anything if left blank.

I can take out this case for now and add it back when custom target is implemented?

Reply:
Oh, interesting. Up to you whether to leave it or not, I was just curious

Reply:
I'll remove it for now. No need to make things more complicated I guess.